### PR TITLE
Adds style for links in not found pages

### LINKED ISSUE #827: Minor bug fix: default-styled links
**Describe the bug**
Some links have browser default appearnce (blue underlined), we should apply algorea styles to them.
For instance, see the error message links on [this page](https://dev.algorea.org/en/#/activities/by-id/1/details)

**Screenshots**
image

## Goals

1. Find all the links with default style (probably those without the `'alg-link'` class name)
2. Apply algorea styles to each of those links.

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes #827

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

- [ ] Case 1:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/links-default-style/en/#/activities/by-id/000;path=4702;parentAttempId=0/details)
  4. Then I see styled link "home page"

- [ ] Case 2:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/links-default-style/en/#/not-found)
  4. Then I see styled link "home page"

### CODE REVIEW

File: src/app/modules/item/pages/item-by-id/item-by-id.component.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
   <p class="alg-error-message" [style.visibility]="(backToHome.isActive)?'hidden':'visible'">
     <span i18n>Go back to the </span>
     <a
+      class="alg-link base-color"
```
Comment (ThomasJuster) [inline src/app/modules/item/pages/item-by-id/item-by-id.component.html:14]: The issue is completely addressed, however in [this page](https://dev.algorea.org/branch/feature/links-default-style/en/#/activities/by-id/1121212121212121/details), the not found message style is weird, the top line should have the same font and be centered as well, with also the same font-size.
Considering this PR is very small I think we can fix the style of "Error while loading the item"
image

File: src/app/core/pages/page-not-found/page-not-found.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <p i18n>This page does not exist!</p>
 <p>
   <span i18n>Go back to the </span>
-  <a routerLink="/" i18n>home page</a>
```
Comment (ThomasJuster) [inline src/app/core/pages/page-not-found/page-not-found.component.html:4]: I noticed that the not found page hides the left nav, it's due to another task of mine. Can you add the following to `page-not-found.component` :pray: :
```ts
  constructor(
    private layoutService: LayoutService,
  ) {
    this.layoutService.configure({ fullFrameActive: false });
  }
```

image

File: src/app/modules/item/pages/item-by-id/item-by-id.component.html (hunk 1 above)
Comment (Iloveall) [reply src/app/modules/item/pages/item-by-id/item-by-id.component.html:14]: Fixed

File: src/app/core/pages/page-not-found/page-not-found.component.html (hunk 2 above)
Comment (Iloveall) [reply src/app/core/pages/page-not-found/page-not-found.component.html:4]: Fixed

**Review by ThomasJuster — COMMENTED** [review]
Fine by me; thanks for the changes :pray:

**Review by smadbe — APPROVED** [review]

**Pull request merged**